slider/curve.py

```diff
@@ -39,18 +40,12 @@ def __call__(self, t):
         raise NotImplementedError('__call__')
 
     def __init_subclass__(cls):
-        cls._kind_dispatch
-        for meth in super(cls, cls).__abstractmethods__:
-            impl = getattr(cls, meth)
-            if impl.__doc__ is None:
-                impl.__doc__ = getattr(super(cls, cls), meth).__doc__
-
         for kind in cls.kinds:
             cls._kind_dispatch[kind] = cls
 
 
 class Bezier(Curve):
-    kinds = 'L'
+    kinds = ()
```

Comment:
I normally use the unit tuple for empty sequences. Strings are used elsewhere because they have the nice property that they are iterables of characters. To be more clear we could make the other kinds tuples as well. What do you think?